packages/@biomejs/biome/bin.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh
+set -eu
+
+CURRENT_DIR=$(dirname "$0")
+IS_X_CALL=false
+CACHE_DIR="."
+
+BINARY_NAME='biome'
+BINARY_NAME_SCOPE='@biomejs/cli-'
+
+# Test if calling via `npx` or `bunx`
+if "$(echo pwd)" | grep -q ".bun/install/cache"; then
+  IS_X_CALL=true
+  CACHE_DIR=$(bun pm cache ls)
+elif "$(echo pwd)" | grep -q ".npm/_npx"; then
```

Comment:
```suggestion
elif "$(echo pwd)" | grep -q '.npm/_npx'; then
```

Reply:
Other than formatting, it does not change nothing at code